### PR TITLE
Patch/My Timeline default UI

### LINKED ISSUE #519: Use 24h format for MyTime times
Now:
We use AM and PM to show hours, 
- 4:00 AM
- 4:00 PM

Wanted:
Use 24h format: 
- 4:00
- 16:00

### LINKED ISSUE #520: Today uncollapsed by default in MyTime
The current day should be uncollapsed every time the MyTime page is openened.

It is more probable that you want to see the Today schesule than the Today summary. Also, for first time users, the schedule is a concept more familiar (and thus less confusing) than the summary.

### PR DESCRIPTION
**Pull request by Tushar-4781**
Resolves #519 
Resolves #520

### CODE REVIEW

File: src/components/MyTimeComponents/MyTimeline.tsx [hunk 1]
```diff
@@ -17,15 +17,17 @@ export const MyTimeline = ({ myTasks }: {myTasks: GoalItem[]}) => {
...
         return (
           <>
             <button
               type="button"
               onClick={() => handleDisplayOptions(task)}
               className="MTL-startTime"
             >
-              {time?.slice(0, -6)} {time?.slice(-2)}
+              {`${Math.floor(hrs / 10) === 0 ? 0 : ""}${hrs}:`}
+              {`${Math.floor(mins / 10) === 0 ? 0 : ""}${mins}`}
```
Comment (tijlleenders) [inline src/components/MyTimeComponents/MyTimeline.tsx:30]: Why are we doing custom logic here?
Date.getHours() and Date.getMinutes() should already give the required info.

Comment (Tushar-4781) [reply src/components/MyTimeComponents/MyTimeline.tsx:30]: @tijlleenders Actually, it's for keeping the hrs and minutes in 2 digits. For e.g., if the time 05:15 then without this logic the getHours() will give us 5:15 so to keep things similar I wrote this logic. In-built function returns number

Comment (tijlleenders) [reply src/components/MyTimeComponents/MyTimeline.tsx:30]:
```suggestion
              {task.start?.toTimeString().slice(0, 5)};
```

Comment (tijlleenders) [reply src/components/MyTimeComponents/MyTimeline.tsx:30]: This works and is easier to read the intent from.
Definition of hrs and mins earlier in code can then also be deleted.

File: src/components/MyTimeComponents/MyTimeline.tsx [hunk 2]
```diff
@@ -17,15 +17,16 @@ export const MyTimeline = ({ myTasks }: {myTasks: GoalItem[]}) => {
     <div id="MTL-times">
       {vbarUp && <div className="bar" />}
       {tasks.map((task: GoalItem) => {
-        const time = task.start?.toLocaleTimeString();
+        const hrs = task.start?.getHours() || 0;
+        const mins = task.start?.getMinutes() || 0;
```
Comment (tijlleenders) [inline src/components/MyTimeComponents/MyTimeline.tsx:21]:
```suggestion
```

Comment (tijlleenders) [reply src/components/MyTimeComponents/MyTimeline.tsx:21]: Remove unused variables

Comment (Tushar-4781) [reply src/components/MyTimeComponents/MyTimeline.tsx]: added the commit

**Review by tijlleenders — APPROVED** [review]

**Pull request merged**